### PR TITLE
Provide sanity test to check we are consistently publishing instant articles to Facebook.

### PR DESCRIPTION
**Pull request by LATaylor-guardian**

### CODE REVIEW

**Comment by LATaylor-guardian** [comment]
Have updated the Team City config for CODE so that it has the facebook instant article host.

File: app/com/gu/contentapi/sanity/FacebookInstantArticlesIsPublishingTest.scala [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+class FacebookInstantArticlesIsPublishingTest(context: Context) extends SanityTestBase(context) {
+
+  "Facebook instant articles" should "have published something in the last 15 minutes" in {
+    val httpRequest = WS.url(s"${Config.facebookInstantArticlesHost}healthcheck/publication").withRequestTimeout(10000.millis).get()
+    whenReady(httpRequest) { result =>
+      assume(result.status == 200, "Cannot access publication healthcheck endpoint for FB IA")
+      val json = Json.parse(result.body)
+      (json \ "lastPublished").toOption.map{ lastPublished =>
+        val lastPublishedDateTime = new DateTime(lastPublished.toString)
+        Minutes.minutesBetween(lastPublishedDateTime, new DateTime()).isLessThan(Minutes.minutes(15)) should be(true)
```
Comment (mchv) [inline app/com/gu/contentapi/sanity/FacebookInstantArticlesIsPublishingTest.scala]: Are you sure we reach this frequency during night in UK times?

Comment (LATaylor-guardian) [inline app/com/gu/contentapi/sanity/FacebookInstantArticlesIsPublishingTest.scala]: Not sure so thought I'd put the PR up to discuss!

**Comment by mchv** [comment]
👍  just need to ensure this the right threshold, but we can try as it is!

File: app/com/gu/contentapi/sanity/FacebookInstantArticlesIsPublishingTest.scala (hunk 1 above)
Comment (LATaylor-guardian) [inline app/com/gu/contentapi/sanity/FacebookInstantArticlesIsPublishingTest.scala]: 30 minutes may be more reasonable.

Comment (mchv) [inline app/com/gu/contentapi/sanity/FacebookInstantArticlesIsPublishingTest.scala]: Let's try with `30 minutes` and check that we are receiving an alert during the night!
😴

Comment (LATaylor-guardian) [inline app/com/gu/contentapi/sanity/FacebookInstantArticlesIsPublishingTest.scala]: I've set it to 30 minutes but will deploy tomorrow morning to spare a noisy night!

**Pull request merged**